src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
 
   def generate_FDB_1_default_config(self, filename):
     """Generates the WinnForum configuration for FDB.1"""
-    # Load device info
-    device_c = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_c.json')))
-    # Send grant request
-    grant_0 = json.load(
-      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-
-    exclusion_zone_db = json.load(open(os.path.join('testcases', 'testdata', 'exclusion_zone_db.json')))
-
-    # Create the actual config.
-    config = {
-        'fccIds': [(device_c['fccId'], 47)],
-        'userIds': [device_c['userId']],
-        'registrationRequests': [device_c],
-        'grantRequests': [grant_0],
-        'expectedResponseCodes': [0],
-        'expectedGrantResponseCodes': [0,400],
-        'expectedHBResponseCodes':[500,501],
-        'expectedRelinquishResponseCodes':[0],
-        'exclusion_zone_db':exclusion_zone_db
-
-    }
-    writeConfig(filename, config)
-
-  def generate_FDB_2_default_config(self, filename):
-    """Generates the WinnForum configuration for FDB.2"""
-    # Load device info
-    device_c = json.load(
+    #Load device info
+    device_a = json.load(
         open(os.path.join('testcases', 'testdata', 'device_a.json')))
-    # Send grant request
+    #Send grant request
     grant_0 = json.load(
       open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-
-    dpa_movelist_db = json.load(open(os.path.join('testcases', 'testdata', 'dpa_db.json')))
-
-    # Create the actual config.
+    #Frequency range 'F2' to modify the exclusion zone database
+    frequency_f2_exz = { 'lowFrequency' : 3680000000,
+                        'highFrequency' : 3690000000 
+                       }
+    #Dictionary containing database information  
+    fake_fcc_db = { 'name': 'Fake_FCC_DB',
+                    'url': 'http://localhost:9090/fakedatabase/exclusionZone',
+                    'databaseFile': 'fakedatabase/exclusion_zone_db.json'
+                  }
+    #Reading the Exclusion zone sample record 
+    data = json.load(
+      open(os.path.join('testcases', 'testdata', 'exz_record_0.json')))
+    #Creating Exclusion zone database
+    writeDB(fake_fcc_db['databaseFile'], data)
```

Comment:
The DB file should be part of the config, not saved to disk during config generation. (It is OK to save to disk during test case execution, but the files should be cleaned up afterward.)

Reply:
Moved creation of Exclusion zone database (exclusion_zone_db.json) from   config generation to test case definition.      Removing the database file ("exclusion_zone_db.json") at the end   of test case execution